package.json

```diff
@@ -53,7 +57,8 @@
     "style-dictionary"
   ],
   "exports": {
-    ".": "./index.js"
+    "import": "./index.js",
+    "require": "./index.cjs"
```

Comment:
not 100% sure about this code, but looks fine given my current knowledge

Reply:
Nope shouldn't conflict with that afaik.